package.json

```diff
@@ -2,61 +2,77 @@
   "name": "bind-event-listener",
   "version": "3.0.0",
   "private": false,
-  "description": "Making binding and unbinding DOM events easier",
+  "type": "module",
+  "description": "Safe DOM Event binding and unbinding",
   "author": "Alex Reardon <alexreardon@gmail.com>",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/alexreardon/bind-event-listener.git"
   },
   "keywords": [
+    "event listener",
+    "event handler",
+    "dom events",
     "dom",
     "ui",
-    "event listener",
-    "event handler"
+    "bind"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
   "sideEffects": false,
+  "main": "./dist/cjs/index.js",
```

Comment:
who's the target of this field if you have `type: "module"` + `exports` pointing to an ESM file?

Reply:
I think I am exposing my lack of understanding on what things I should be doing to make sure everybody has a good time 😅